docs/resie_energy_system_components.md

```diff
@@ -345,198 +347,223 @@ Symbol | Description | Unit
 
-$$ \eta_{HEL,H_2} = \eta_{HEL,H_2,start} + \left ( \frac{\delta_{HEL,H_2}}{100} \ {\frac{FLH}{10.000}} \right )$$
-$$ \eta_{HEL,heat} = \eta_{HEL,heat,start} + \left ( \frac{\delta_{HEL,heat}}{100} \ {\frac{FLH}{1.000}} \right ) $$
-$$ \text{with } FLH = \frac{\text{total hydrogen produced since start [kg]}}{\text{nominal hydrogen production [kg]}} $$
+$$ P_{losses} = \dot{Q}_{loss,heat} - P_{H_2,loss} $$
 
-Keep in mind: Test if sum of efficiencies will reach values > 1 or if one efficiency falls below zero for the given maximum changing intervale of the stacks (ToDo)
+Since the oxygen produced during the electrolysis process can also be utilized under certain circumstances, the resulting oxygen mass flow \(\dot{m}_{O_2,out}\) is determined from the stoichiometric ratio of the hydrolysis reaction equation:
 
-With a known mass flow \(\dot{m}_{HP,cool}\) and the specific heat capacity of the heat transfer medium of the heat recovery \(c_{p,cool}\) as well as a known inlet temperature \(T_{HEL,cool,in}\), the outlet temperature of the heat transfer medium from the cooling circuit \(T_{HEL,cool,out}\) can be determined by rearranging the following equation:
-$$\dot{Q}_{HEL,waste heat} = \dot{m}_{HP,cool} \enspace c_{p,cool} \enspace (T_{HEL,hot,out} - T_{HEL,cool,in})$$
+$$ \dot{m}_{O_2,out} =  v_{O_2,H_2} \ \dot{m}_{H_2} $$
+$$ \text{with} \quad v_{O_2,H_2} = \frac{atomic \ mass \ O_2}{2 \cdot atomic \ mass \ H_2} = \frac{2 \cdot 15.999 \ u}{2 \cdot 2 \cdot 1.0008 \ u} = 7.9386 $$
 
-The heat loss \(\dot{Q}_{HEL,loss}\), which cannot be used and is dissipated to the environment via heat transport mechanisms, is calculated as follows
+**Note: At the moment the pressure, at which the stacks operate and the hydrogen is extracted and purified, is assumed to be constant throughout the entire process and does not affect efficiencies or heat by-products. This might be addressed in a future release.**
 
-$$  \dot{Q}_{HEL,loss} = P_{el,HEL} - \dot{Q}_{HEL,waste heat} - \dot{E}_{HEL,H_2} \\ =  P_{el,HEL} \ (1 - \eta_{HEL,heat} + \eta_{HEL,heat} \ \eta_{HEL,H_2} - \eta_{HEL,H_2}) $$
+### Unit dispatch
 
-The actual needed power supply of the electrolyser \(P_{el,HEL,supply}\) increases by losses in the power electronics and results from the electrical reference power \(P_{el,HEL}\) and the losses in the power electronics \(\eta_{HEL,PE}\) to
-$$ P_{el,HEL,supply} = \frac{P_{el,HEL}}{\eta_{HEL,PE}} $$
+Depending on the sizing and technology of realised electrolysers, the whole plant often consists of more than one stack and/or more than one set of power supply equipment. This is modeled as the electrolyser consisting of \(N_{unit}\) units, which are all the same in regards to design power and efficiencies. The efficiency functions given as input parameters thus relate to a single unit with its own power supply subsystem. The inputs are outputs over all units active in a single timestep are summed together with no losses between the units.
```

Comment:
"The inputs are outputs over all units active in a single timestep are summed together with no losses between the units."
What is meant by this sentence?